nvm.sh

```diff
@@ -700,20 +700,24 @@ nvm_ls_remote_iojs() {
   echo "$VERSIONS"
 }
 
+nvm_checksig() {
+    if nvm_has 'gpg'; then
+        if ! LC_ALL=C gpg --verify $1; then
+            return 1
+        fi
+    else
+        echo "You should install GnuPG to verify the authenticity of the archives to download: https://www.gnupg.org/" >&2
+    fi
+}
+
 nvm_checksum() {
-  local NVM_CHECKSUM
-  if nvm_has "sha1sum" && ! nvm_is_alias "sha1sum"; then
```

Comment:
@ljharb How useful is checksum checking without checking the signatures? ([ref](https://github.com/jchaney/owncloud/pull/12#issuecomment-99544307))